src/core/LiterallyCanvas.coffee

```diff
@@ -1,3 +1,4 @@
+React = require '../reactGUI/React-shim'
```

Comment:
Why would you need to import it in the core/LiterallyCanvas file? This part of the program should be independent of the UI.

Reply:
Good catch. Indeed not needed in src/core/LiterallyCanvas.coffee.
Did gulp + npm pack and tested it in my application.